actions/send_embed_message.js

```diff
@@ -119,7 +116,7 @@ module.exports = {
       target.send(messageContent, { embed })
         .then((msg) => {
           if (msg && varName3) this.storeValue(msg, storage3, varName3, cache)
-          this.callNextAction(cache)
+          return this.callNextAction(cache)
```

Comment:
This doesn't change anything

```suggestion
          this.callNextAction(cache)
```